### DISCUSSION TITLE
Motion Sensor works but no communication

### DISCUSSION THREAD

**Discussion by Nick4-1** [Koenkk/zigbee2mqtt#17463] [Devices]
Hi, HA user for a few years, started with zigbee 1y ago, switched to z2m a few months now.

My network works fine, no problems but...
I have a motion sensor (SNZB-03, I know, they are not the best) and apart from having false positives I have no issues.
Now al of a sudden the light doesn't come on anymore with motion.
Battery: 51,5%
LQI: 111

Since I'm fairly new with z2m I would like to know how to troubleshoot this instead of removing/pairing again - if there are other options of course.
Nothing in the logs for this device so I have no idea what might be wrong or how solve this.
All other devices work as normal.

Thanks for any help which is highly appreciated!

**Comment by lux73** [comment]
first try -> delete this Sensor form z2m (force remove in WebIf) and try to completely new pair with z2m

btw. some Devices would not work anymore with such an low Battery (the Battery Calculating is not the best in z2m atm) - try to replace the Battery with new
